pkgs/development/python-modules/Theano/default.nix

```diff
@@ -68,7 +68,9 @@ in buildPythonPackage rec {
       --replace 'StrParam(default_dnn_base_path)' 'StrParam('\'''${cudnn}'\''')'
   '';
 
-  preCheck = ''
+  # needs to be postFixup so it runs before pythonImportsCheck even when
```

Comment:
Did you mean preFixup here?

Reply:
Bah. Well spotted. The comment's right, code's wrong. Fixed.